File: app/src/main/java/it/unical/mat/embasp/languages/pddl/PDDLInputProgram.java
package it.unical.mat.embasp.languages.pddl;

import it.unical.mat.embasp.base.InputProgram;

public class PDDLInputProgram extends InputProgram {

	private final PDDLProgramType programsType;

	public PDDLInputProgram(final PDDLProgramType type) {
		super();
		programsType = type;
	}

	public PDDLProgramType getProgramsType() {
		return programsType;
	}

}

File: CSharp version/it/unical/mat/embasp/languages/pddl/PDDLInputProgram.cs
namespace it.unical.mat.embasp.languages.pddl
{
    using InputProgram = it.unical.mat.embasp.@base.InputProgram;

    public class PDDLInputProgram : InputProgram
    {
        private readonly PDDLProgramType programsType;

        public PDDLInputProgram(PDDLProgramType type) : base() => programsType = type;

        public virtual PDDLProgramType ProgramsType => programsType;
    }
}